Dockerfile

```
@@ -32,8 +32,14 @@ WORKDIR /app
 
 # Install gettext
 RUN apt-get update && \
-  apt-get install -y \
-  gettext && \
+     apt-get install -y \
+      gettext \
+      libcairo2 \
+      libffi-dev \
+      libgdk-pixbuf2.0-0 \
+      libpango-1.0-0 \
+      libpangocairo-1.0-0 \
+      shared-mime-info && \
```

Just an extra space to remove
```suggestion
    apt-get install -y \
      gettext \
      libcairo2 \
      libffi-dev \
      libgdk-pixbuf2.0-0 \
      libpango-1.0-0 \
      libpangocairo-1.0-0 \
      shared-mime-info && \
```